api/applications_api.py

```diff
@@ -67,6 +73,9 @@ class Meta:
             "year_established": {
                 "help_text": "Year when the organisation was established.",
             },
+            "active_members": {
+                "help_text": "Active space user person count of the organization.",
```

Comment:
Active space user person count?

Reply:
:laughing:  yes
I tried to point out perhaps the obvious and surely sounds like a "rabbit hole". 
Perhaps just  "Number of active persons in the organization"?

Reply:
Yeah, that sounds good :)

Reply:
Fix'd